modules/virt-editing-cdi-cpu-and-memory-defaults.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-cdi-for-namespace-resourcequota.adoc
+
+[id="virt-editing-cdi-cpu-and-memory-defaults_{context}"]
+= Editing the `CDIConfig` object to override CPU and memory defaults
+
+Modify the default settings for CPU and memory requests and limits for your
+use case by editing the spec attribute of the `CDIConfig` object.
+
+.Procedure
+
+. Use the `oc` client to edit the `cdiconfig/config` and edit the
+`spec: podResourceRequirements` to change the default CPU and memory
+requests/limits.
++
+----
+$ oc edit cdiconfig/config
+----
+
+. Update the `spec` property of the `CDIConfig` object
++
+[source,yaml]
+
+----
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: CDIConfig
+metadata:
+  labels:
+    app: containerized-data-importer
+    cdi.kubevirt.io: ""
+  name: config
+spec:
+    podResourceRequirements:
+    limits:
+      cpu: "4"
+      memory: "1Gi"
+    requests:
+      cpu: "1"
+      memory: "250Mi"
+...
+----
+
+. Save and exit the editor to update the `CDIConfig` object.
+
+. Use the `oc` client to retrieve the `CDIConfig` status and verify your changes.
```

Comment:
You can add this under a `.Verification steps` section, and as mentioned earlier, I'd phrase the step more like "View the `CDIConfig` status by running the following command:"